src/main/java/net/openhft/chronicle/wire/channel/ChronicleChannel.java

```diff
@@ -88,7 +88,8 @@ default <T> T readOne(StringBuilder eventType, Class<T> expectedType) throws Clo
             try (DocumentContext dc = readingDocument()) {
                 if (dc.isPresent()) {
                     ValueIn in = dc.wire().read(eventType);
-                    if (StringUtils.isEqual(eventType, MethodReader.HISTORY)) {
+                    if (StringUtils.isEqual(eventType, MethodReader.HISTORY) ||
+                            StringUtils.isEqual(eventType, "" + MethodReader.MESSAGE_HISTORY_METHOD_ID)) {
```

Comment:
I assume this gets optimised into a constant by the compiler, if not perhaps we should make it a constant?

Reply:
It should as it's known at compile time.